Documentation/network/concepts/ipam/cluster-pool.rst

```diff
@@ -45,6 +45,13 @@ Configuration
 For a practical tutorial on how to enable this mode in Cilium, see
 :ref:`gsg_ipam_crd_cluster_pool`.
 
+.. note::
+    An existing element of the ``clusterPoolIPv4PodCIDRList`` list shouldn't be removed or changed. 
+    Doing so will cause unexpected behaviour. If you face a situation where the pool is exhausted,
+    add a new element to the list. The minimum mask length is ``/30``, but a mask of at least ``/29`` 
+    is recommended. The reason for this is that the allocator will reserve 2 IPs per CIDR block for 
+    the network and broadcast addresses. Changing ``clusterPoolIPv4MaskSize`` is also not supported.
```

Comment:
Use present tense, use US English, use clear, plain language
```suggestion
    Don't change any existing elements of the ``clusterPoolIPv4PodCIDRList`` list, as
    changes cause unexpected behavior. If the pool is exhausted,
    add a new element to the list. The minimum mask length is ``/30``, with a recommended minimum mask 
    length of at least ``/29``. The reason to add new elements rather than change existing elements is that
    the allocator reserves 2 IPs per CIDR block for 
    the network and broadcast addresses.
    
    Changing ``clusterPoolIPv4MaskSize`` is also not recommended.
```

Reply:
Thank you very much for the correction.